bootstrap.sh

```diff
@@ -1,48 +1,383 @@
-#!/bin/bash
-# Simple bootstrap script to build and run the daemon
+#!/bin/sh
+# Originally created by FeralInteractive <https://www.feralinteractive.com/en/> under BSD 3-Clause License <https://github.com/FeralInteractive/gamemode/blob/master/LICENSE.txt> in 2017 bootstrap.sh
+# Performed full refact by Jacob Hrbek <kreyren@rixotstudio.cz> under GPLv3 license <https://www.gnu.org/licenses/gpl-3.0.en.html> in 05.07.2020 04:28:55 CET
 
-if [ "$EUID" -eq 0 ]
-  then echo "Please don't run bootstrap.sh as root."
-  exit
-fi
+# shellcheck shell=sh # Written to be POSIX-compatible
+
+###! Simple bootstrap script to build and run the daemon
+###! We need:
+###! - FIXME-DOCS
+###! Requires:
+###! - FIXME
+###! Exit codes:
+###! - FIXME-DOCS(Krey): Defined in die()
+###! Platforms:
+###! - [ ] Linux
+###!  - [ ] Debian
+###!  - [ ] Ubuntu
+###!  - [ ] Fedora
+###!  - [ ] NixOS
+###!  - [ ] Archlinux
+###!  - [ ] Alpine
+###! - [ ] FreeBSD
+###! - [ ] Darwin
+###! - [ ] Redox
+###! - [ ] ReactOS
+###! - [ ] Windows
+###! - [ ] Windows/Cygwin
+
+# Maintainer info
+UPSTREAM="https://github.com/FeralInteractive"
+MAINTAINER="https://github.com/Kreyren"
```

Comment:
It's nice that you want to maintain this file, but I think that's something you should ask beforehand since you don't have write access to this repo.